src/main/java/frc/robot/subsystems/CoralGripper.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.spark.SparkLowLevel;
+import com.revrobotics.spark.SparkMax;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+import org.ironmaple.simulation.drivesims.COTS;
+
+public class CoralGripper extends SubsystemBase {
+
+
+    private SparkMax motor;
+    private DigitalInput limitSwitch;
+
+
+    public CoralGripper() {
+        motor = new SparkMax(RobotMap.MOTOR_IDENTIFIER, SparkLowLevel.MotorType.kBrushless);
+        limitSwitch = new DigitalInput(1);
```

Comment:
```suggestion
        limitSwitch = new DigitalInput(RobotMap.CORAL_GRIPPER_LIMIT_SWITCH);
```
Ports/IDs/Connections are to be defined in `RobotMap`